### PR TITLE
add FAQ about how to request an increase of quota

### PR DESCRIPTION
**Pull request by lexming**

### CODE REVIEW

**Review by wpoely86 — APPROVED** [review]

File: source/access/how_to_request_more_quota.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  in the `Edit VO <https://account.vscentrum.be/django/vo/edit>`_ tab of the VSC
+  account page.
+
+Requesting more storage space
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+VO moderators can request additional quota for ``VSC_DATA_VO`` and ``VSC_SCRATCH_VO``:
+
+#. Go to the section **Request additional quota** in the
+   `Edit VO <https://account.vscentrum.be/django/vo/edit>`_ page
```
Comment (smoors) [inline source/access/how_to_request_more_quota.rst:57]:
```suggestion
   `Edit VO <https://account.vscentrum.be/django/vo/edit>`_ tab
```

**Review by smoors — APPROVED** [review]
lgtm

File: source/access/managing_disk_usage.rst [hunk 2]
```diff
@@ -1,22 +1,43 @@
...
-using the `web application <https://account.vscentrum.be/>`_.
+VSC account page
+----------------
+
+Your VSC account page shows up-to-date information about your storage usage on
+each of the file systems accessible to your account. This information is
+available in the *Usage* section of the
+`View Account <https://account.vscentrum.be>`_ tab and includes:
+
+* total data size
```
Comment (stevenvdb) [inline source/access/managing_disk_usage.rst:24]: I'm not sure we (at KU Leuven) have set this up, for my personal vsc account the Used column is n/a.

File: source/access/how_to_request_more_quota.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  If that is not the case or if in doubt, please contact :ref:`user support
+  <tech support VSC>`.
+
+VSC_SCRATCH
+  On VSC sites that support :ref:`Virtual Organizations (VO)
+  <virtual_organization>`, it is preferable to join a VO to get access to a
+  larger ``VSC_SCRATCH_VO``.
+  If that is not the case or if in doubt, please contact :ref:`user support
+  <tech support VSC>`.
+
```
Comment (stevenvdb) [inline source/access/how_to_request_more_quota.rst:40]: Would it be useful to have site-specific links here? We have a specific page in our service catalog, personally I would prefer our users to look there.

File: source/access/managing_disk_usage.rst (hunk 2 above)
Comment (lexming) [reply source/access/managing_disk_usage.rst:24]: Can you verify if that is intended or just a bug on your side?

File: source/access/how_to_request_more_quota.rst (hunk 3 above)
Comment (lexming) [reply source/access/how_to_request_more_quota.rst:40]: Sure, I'll add your link. Do you have a scpecific URL for this? or should I link to https://icts.kuleuven.be/sc/HPC?

File: source/access/managing_disk_usage.rst (hunk 2 above)
Comment (stevenvdb) [reply source/access/managing_disk_usage.rst:24]: It seems we don't send any data about this. We will see internally if we can set this up, for now you can ignore this comment.

File: source/access/how_to_request_more_quota.rst (hunk 3 above)
Comment (stevenvdb) [reply source/access/how_to_request_more_quota.rst:40]: If possible, link to https://icts.kuleuven.be/sc/onderzoeksgegevens/HPC-storage mentioning this is specific for KU Leuven users.

Comment (smoors) [reply source/access/how_to_request_more_quota.rst:40]: while you're at it, can you also add a link to https://hpc.vub.be/docs/vo/#requesting-more-storage-space for VUB users?

File: source/access/how_to_request_more_quota.rst [hunk 4]
```diff
@@ -87,3 +92,12 @@ maximally use. By default, this is set to 50% of the total quota for each user.
    for any user indicates what he/she can maximally use, but the actual limit
    will then depend on the usage of the other members. The total storage quota
    of the VO will always be respected.
+
+Tier-1 project
+==============
+
+The scratch storage of your project in tier-1 is managed by the institute
+hosting the tier-1 HPC infrastructure. In general, your project will get as
+much quota as requested in your application and that should be sufficient to
```
Comment (smoors) [inline source/access/how_to_request_more_quota.rst:101]: they don't always get what they ask for ;)
```suggestion
hosting the tier-1 HPC infrastructure. In general, your project directory will get as
much quota as granted to your project and that should be sufficient to
```

Comment (lexming) [reply source/access/how_to_request_more_quota.rst:101]: That's why I said _In general_, as in _not always_.

Comment (smoors) [reply source/access/how_to_request_more_quota.rst:101]: that's confusing. there is no general rule, the allocation board decides how much you get.

**Pull request merged**